about.adoc

```diff
@@ -19,6 +19,5 @@
 | SMBIOS          | System Management BIOS cite:[SMBIOS].
 | SoC             | System on a chip, a combination of processor and supporting chipset logic in single package.
 | System          | A system is the entirety of a computing entity, including all elements in a platform (hardware, firmware) and software (hypervisor, operating system, user applications, user data). A system can be thought of both as a logical construct (e.g. a software stack) or physical construct (e.g. a notebook, a desktop, a server, a network switch, etc).
-| TAD             | Time and Alarm Device cite:[ACPI] Section 9.17.
```

Comment:
How is this insignificant, when the ACPI section talks about TAD?

Reply:
TAD is written exactly twice in the entire document.  First, where it is defined inline, and second, in the latter half of **the** **same** **sentence**.  Thus, insignificant.

MADT, PPTT, MCFG, ECAM, RINTC, SPCR, AML, UART, and PCIe are also missing from the glossary, among others. Shall we copy the entire ACPI and UEFI glossary into the BRS?

Reply:
It would be reasonable to cover those in the glossary, yes.

Reply:
On a second thought, yes TAD appears to be insignificant because it is used and defined in the same section only.
MADT, PPTT, RINTC, SPCR,   fit this pattern too.

ECAM really needs to be defined in line.

Reply:
Proposed separately via rhttps://github.com/riscv-non-isa/riscv-brs/pull/59